frontend/lib/norent/components/social-icons.tsx

```diff
@@ -17,15 +17,19 @@ const socialMediaPageLinks = [
 
 const prefilledTwitterCopy = () =>
   li18n._(
-    t`No idea how you'll pay rent this month? Tell your landlord with norent.org from @JustFixNYC. This free tool sends a certified letter informing them of your protections. Join the #cancelrent movement at norent.org.`
+    t(
+      "norent.tweetTemplateForSharingNoRent"
+    )`No idea how you'll pay rent this month? Tell your landlord with norent.org from @JustFixNYC. This free tool sends a certified letter informing them of your protections. Join the #cancelrent movement at norent.org.`
```

Comment:
Oh super dope, I didn't realize there was a way to manually specify an ID for these non-JSX thingys!